automation/service/workflow_step_args.go

```diff
@@ -0,0 +1,71 @@
+package service
+
+import (
+	"context"
+	"github.com/cortezaproject/corteza-server/automation/types"
+	"github.com/cortezaproject/corteza-server/pkg/expr"
+)
+
+type (
+	// argsProc is a helper to process arguments
+	//
+	// when initialised it evaluates expressions using given scope
+	// it provides methods to extract values from the result
+	argsProc struct {
+		expr   types.ExprSet
+		result *expr.Vars
+	}
```

Comment:
would it make sense to add some tests for this?

Reply:
It would make a lot of sense yes. 
And since this is a utility funct/struct it does not make sense (for me) to write tests so early